### PR TITLE
Cleanup cards

### PR DESCRIPTION
**Pull request by iantrich**

### CODE REVIEW

**Comment by thomasloven** [comment]
I pulled this and every card seems to work, at least...

File: src/panels/lovelace/cards/hui-markdown-card.ts [hunk 1]
```diff
@@ -64,35 +61,39 @@ export class HuiMarkdownCard extends LitElement implements LovelaceCard {
...
-          color: var(--primary-color);
-        }
-        ha-markdown img {
-          max-width: 100%;
-        }
-      </style>
+  static get styles(): CSSResult {
+    return css`
+      :host {
+        @apply --paper-font-body1;
```
Comment (balloob) [inline src/panels/lovelace/cards/hui-markdown-card.ts]: We can remove this. This has never worked.

Comment (balloob) [reply src/panels/lovelace/cards/hui-markdown-card.ts]: `@apply` is a deprecated spec that Polymer supports, but will never become a standard.

File: src/panels/lovelace/cards/hui-picture-entity-card.ts [hunk 2]
```diff
@@ -132,37 +129,39 @@ class HuiPictureEntityCard extends LitElement implements LovelaceCard {
...
+        overflow: hidden;
+        position: relative;
+      }
+
+      hui-image.clickable {
+        cursor: pointer;
+      }
+
+      .footer {
+        @apply --paper-font-common-nowrap;
```
Comment (balloob) [inline src/panels/lovelace/cards/hui-picture-entity-card.ts]: This won't work.

File: src/panels/lovelace/cards/hui-picture-glance-card.ts [hunk 3]
```diff
@@ -177,44 +176,47 @@ class HuiPictureGlanceCard extends LitElement implements LovelaceCard {
...
+        min-height: 48px;
+        overflow: hidden;
+      }
+
+      hui-image.clickable {
+        cursor: pointer;
+      }
+
+      .box {
+        @apply --paper-font-common-nowrap;
```
Comment (balloob) [inline src/panels/lovelace/cards/hui-picture-glance-card.ts]: This won't work.

File: src/panels/lovelace/cards/hui-entities-card.ts [hunk 4]
```diff
@@ -128,38 +127,42 @@ class HuiEntitiesCard extends LitElement implements LovelaceCard {
...
+      #states > * {
+        margin: 8px 0;
+      }
+
+      #states > div > * {
+        overflow: hidden;
+      }
+
+      .header {
+        @apply --paper-font-headline;
```
Comment (balloob) [inline src/panels/lovelace/cards/hui-entities-card.ts]: Won't work.

File: src/panels/lovelace/cards/hui-entities-card.ts [hunk 5]
```diff
@@ -128,38 +127,42 @@ class HuiEntitiesCard extends LitElement implements LovelaceCard {
...
-      </style>
+        line-height: 40px;
+        color: var(--primary-text-color);
+        padding: 4px 0 12px;
+        display: flex;
+        justify-content: space-between;
+      }
+
+      .header .name {
+        @apply --paper-font-common-nowrap;
```
Comment (balloob) [inline src/panels/lovelace/cards/hui-entities-card.ts]: Won't work

**Comment by iantrich** [comment]
Going to rename all `Config` interfaces to match card name before merging

**Comment by thomasloven** [comment]
You can't just drop the `@apply` lines.

You have to manually add everything they did, e.g. `font-size: 24px;` for `.header`
skarmavbild 2019-03-01 kl 23 45 40

**Comment by iantrich** [comment]
oh, derp

**Comment by iantrich** [comment]
paper-styles should be copied over now

**Comment by balloob** [comment]
🎉 !!

**Pull request merged**